Documentation/configuring-storage.md

```diff
@@ -56,20 +56,64 @@ The `spec.storage.hive.s3Compatible.secretName` should be the name of a secret i
 
 > **Note**: The values of the `aws-access-key-id` and `aws-secret-access-key` must be base64 encoded.
 
-For example:
-
+Noobaa installation example:
+1. Get noobaa CLI.
+     * Download [noobaa CLI tool](https://github.com/noobaa/noobaa-core) and put it into your bin PATH
+2. Install noobaa into your cluster.
+```bash
+oc create ns noobaa
+oc project noobaa
+noobaa install
+```
+3. After a successful noobaa installation, we create a bucket to store the reporting data.
+```bash
+noobaa bucket create foobar
+```
+4. Extract the AWS access information from the noobaa secret.
+```bash
+$ oc get secret noobaa-admin -n noobaa -o json | jq '.data|map_values(@base64)'
+{
+  "AWS_ACCESS_KEY_ID": "<BASE64_ENCODED_KEY_ID>",
+  "AWS_SECRET_ACCESS_KEY": "<BASE64_ENCODED_ACCESS_KEY>",
+  "email": "<BASE64_ENCODED_EMAIL>",
+  "password": "<BASE64_ENCODED_PASSWORD>",
+  "system": "<BASE64_ENCODED_VALUE>"
+}
+```
+5. Create Secret based on Step 4.
 ```yaml
 apiVersion: v1
 kind: Secret
 metadata:
-  name: your-aws-secret
+  name: noobaa-aws-secret
 data:
-  aws-access-key-id: "dGVzdAo="
-  aws-secret-access-key: "c2VjcmV0Cg=="
+  aws-access-key-id: "<BASE64_ENCODED_KEY_ID_FROM_STEP_4>"
+  aws-secret-access-key: "<BASE64_ENCODED_ACCESS_KEY_FROM_SETP_4>"
```

Comment:
I don't want to hold this up, but it would be nice to have a command that would generate base64 encoding values:

```bash
$ openssl rand -base64 32 | head -c32; echo
```

Reply:
Oh, I misread then example - nevermind.

Reply:
@timflannagan1 yeap, as it stands in the secrets, it's already encoded, so the user doesn't have to do anything but just yank it out from `oc get`